process-engine/process_engine_core
# 🐛 Fix double on error notification

## Changes

1. Fix error evaluation in `ExecuteProcessService`
    - Errors from `EssentialProjects` or BpmnErrors from `ErrorEndEvents` are now rethrown as they are.
2. `ErrorEndEvents` no longer send `onProcessError` notifications to prevent duplicated notifications
    - The `ProcessInstancePersistenceService` already sends these notifications
3. Add `message` to `BpmnErrors`

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/400

PR: #304

## How to test the changes

- Run ProcessModels with ErrorEndEvents
- See that the errors modelled on these events are thrown correctly
- Note that `onProcessError` notifications will only be sent once.
